src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,131 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+import { useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+const tableHead = ['name', 'sex', 'born', 'died', 'mother', 'father'];
+const tableHeadForSort = ['name', 'sex', 'born', 'died'];
+
+interface Props {
+  people: Person[],
+  slug: string,
+}
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  slug,
+}) => {
+  const [searchParams] = useSearchParams();
+
+  const sortBy = searchParams.get('sortBy');
+  const order = searchParams.get('order');
+
+  const sex = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries');
+  const query = (searchParams.get('query') || '').toLowerCase();
+
+  const visiblePeople = useMemo(() => [...people].sort((a, b) => {
+    if (sortBy) {
+      const [personA, personB] = order === 'asc'
+        ? [a, b]
+        : [b, a];
+
+      if (sortBy === 'name' || sortBy === 'sex') {
+        return personA[sortBy].localeCompare(personB[sortBy]);
+      }
+
+      if (sortBy === 'born' || sortBy === 'died') {
+        return personA[sortBy] - personB[sortBy];
+      }
+    }
```

Comment:
use switch case here

Reply:
Marina, hi
Can you help me? I dont understand how to realize switch case here